target/linux/ipq806x/image/Makefile

```diff
@@ -81,6 +81,21 @@ define Device/ZyXELImage
 	IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-to $$$$(BLOCKSIZE) | sysupgrade-tar rootfs=$$$$@ | append-metadata
 endef
 
+define Device/askey_rt4230w
+	$(call Device/LegacyImage)
+	$(call Device/UbiFit)
+	DEVICE_VENDOR := Askey
+	DEVICE_MODEL := RT4230W-REV6
+	SOC := qcom-ipq8065
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	BOARD_NAME := rt4230w
+	DEVICE_PACKAGES := ath10k-firmware-qca9984-ct-full-htt
```

Comment:
Based on the other devices in this target, it looks to me like you should choose ath10k-firmware-qca9984-ct here (without -full-htt). Or do you need the latter for some specific reason?

Reply:
Just needed -ct. That was left over from me trying to figure out why I couldn't change the 5Ghz channel but it turns out it was just taking a bit to bring up the interface.